packages/plugins/releases/package.json

```diff
@@ -0,0 +1,93 @@
+{
+  "name": "@strapi/plugin-releases",
+  "version": "4.14.3",
+  "description": "Strapi plugin for organizing and releasing content",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/strapi/strapi.git",
+    "directory": "packages/plugins/releases"
+  },
+  "license": "SEE LICENSE IN LICENSE",
+  "author": {
+    "name": "Strapi Solutions SAS",
+    "email": "hi@strapi.io",
+    "url": "https://strapi.io"
+  },
+  "maintainers": [
+    {
+      "name": "Strapi Solutions SAS",
+      "email": "hi@strapi.io",
+      "url": "https://strapi.io"
+    }
+  ],
+  "exports": {
+    "./strapi-admin": {
+      "types": "./dist/admin/index.d.ts",
+      "source": "./admin/src/index.ts",
+      "import": "./dist/admin/index.mjs",
+      "require": "./dist/admin/index.js",
+      "default": "./dist/admin/index.js"
+    },
+    "./strapi-server": {
+      "types": "./dist/server/index.d.ts",
+      "source": "./server/src/index.ts",
+      "import": "./dist/server/index.mjs",
+      "require": "./dist/server/index.js",
+      "default": "./dist/server/index.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "files": [
+    "./dist",
+    "strapi-server.js"
+  ],
+  "strapi": {
+    "name": "releases",
+    "description": "Organize and release content",
+    "kind": "plugin",
+    "displayName": "Releases"
+  },
+  "scripts": {
+    "build": "NODE_ENV=production strapi plugin:build --force",
+    "watch": "run -T tsc -w --preserveWatchOutput",
+    "clean": "run -T rimraf ./dist",
+    "prepublishOnly": "yarn clean && yarn build",
+    "test:front": "run -T cross-env IS_EE=true jest --config ./jest.config.front.js",
+    "test:front:watch": "run -T cross-env IS_EE=true jest --config ./jest.config.front.js --watchAll",
+    "test:front:ce": "run -T cross-env IS_EE=false jest --config ./jest.config.front.js",
+    "test:front:watch:ce": "run -T cross-env IS_EE=false jest --config ./jest.config.front.js --watchAll",
+    "test:ts:front": "run -T tsc -p admin/tsconfig.json",
+    "lint": "yarn lint:project && yarn lint:back && yarn lint:front",
+    "lint:project": "run -T eslint . -c ./.eslintrc.js",
+    "lint:back": "run -T eslint ./server -c ./server/.eslintrc.js",
+    "lint:front": "run -T eslint ./admin -c ./admin/.eslintrc.js"
+  },
+  "dependencies": {
+    "@strapi/design-system": "1.12.0",
+    "@strapi/helper-plugin": "4.14.3",
+    "@strapi/icons": "1.12.0",
+    "react-intl": "6.4.1"
+  },
+  "devDependencies": {
+    "@strapi/strapi": "4.14.3",
+    "@testing-library/react": "14.0.0",
+    "@testing-library/user-event": "14.4.3",
+    "@types/styled-components": "5.1.26",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-router-dom": "5.3.4",
+    "styled-components": "5.3.3",
```

Comment:
I think having these in peerDependencies is enough

Reply:
they should be in devDeps even if they're peerDeps otherwise they're considered an erroneous dependency and eslint will complain. It also helps us understand what deps are used in a project correctly imo :)